quickwit/quickwit-proto/src/lib.rs

```diff
@@ -339,7 +339,7 @@ impl IndexUid {
         } else {
             Self (
                 format!("{}:{}", index_id.into(), incarnation_id)
-            )                
+            )
```

Comment:
Interesting. Did you format that manually? It seem that `make fix` doesn't pick that up.

Reply:
My editor is set up to remove trailing whitespaces.